File: review-service/src/main/java/com/example/review_service/model/Review.java
package com.example.review_service.model;
import com.google.cloud.Timestamp;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Review {
    private String reviewId;
    private String movieId;
    private String userId;
    private double rating;
    private String comment;
    private Timestamp createdAt;
    private String userFullName;
}

File: AndroidApp/app/src/main/java/com/btlcnpm/androidapp/data/model/Review.kt
package com.btlcnpm.androidapp.data.model

// Data class cho response từ Review Service (tương ứng Review.java)
data class Review(
    val reviewId: String?,
    val movieId: String?,
    val userId: String?,
    val rating: Double?, // Backend dùng int
    val comment: String?,
    val createdAt: FirestoreTimestamp?, // Backend trả về String (Timestamp.now().toString())
    val userFullName: String?
)